src/components/Anchor/index.js

```diff
@@ -21,13 +21,14 @@ const prefixByType = {
 const Anchor = ({
   url,
   children,
+  className = '',
   isSeparateTab = false,
   type = 'inline',
 }) => (
   <a
     href={url}
     target={isSeparateTab ? '_blank' : '_self'}
-    className={classNameByType[type]}
+    className={`${classNameByType[type]} ${className}`}
```

Comment:
I've found great value in moving layout/wrapping style things to a wrapping elements instead of passing them to components. If a prop is needed then yeah I'd pass it, but if it isn't actually needed in the component but is effecting the outer layout, I'd put it in a generic wrapping div or span.